Mathlib/Analysis/Convex/Deriv.lean

```diff
@@ -583,11 +835,14 @@ lemma slope_lt_of_hasDerivWithinAt_Iio (hfc : StrictConvexOn ℝ S f)
   simp_rw [← slope_def_field, slope_comm _ y] at this
   exact this.trans_le <| hfc.convexOn.slope_le_of_hasDerivWithinAt_Iio hu hy huy hf'
 
-lemma slope_lt_left_deriv (hfc : StrictConvexOn ℝ S f) (hx : x ∈ S) (hy : y ∈ S) (hxy : x < y)
+lemma slope_lt_leftDeriv (hfc : StrictConvexOn ℝ S f) (hx : x ∈ S) (hy : y ∈ S) (hxy : x < y)
```

Comment:
Would you mind doing these renames separately?. This PR is already pretty big.

Reply:
#21076